packages/calendar/src/Calendar.ts

```diff
@@ -0,0 +1,449 @@
+        `;
+    }
+
+    public renderCalendarTableRow(weekIndex: number): TemplateResult {
+        return html`
+            <tr role="row">
+                ${this._getDatesInWeek(weekIndex).map((calendarDate) =>
+                    this.renderCalendarTableCell(calendarDate)
+                )}
+            </tr>
+        `;
+    }
+
+    public renderCalendarTableCell(calendarDate: CalendarDate): TemplateResult {
+        const isOutsideMonth = calendarDate.month !== this._currentDate.month;
+
+        const isSelected = Boolean(
+            this.selectedDate &&
+                isSameDay(this._toCalendarDate(this.selectedDate), calendarDate)
+        );
+
+        const isToday = Boolean(isSameDay(calendarDate, this.today));
+
+        const isDisabled = Boolean(
+            this.disabled ||
+                (this._minDate && calendarDate.compare(this._minDate) < 0) ||
+                (this._maxDate && calendarDate.compare(this._maxDate) > 0)
+        );
+
+        const dayClasses = {
+            'is-outsideMonth': isOutsideMonth,
+            'is-selected': isSelected,
+            'is-today': isToday,
+            'is-disabled': isDisabled,
+        };
+
+        const currentDayTitle = this._formatDate(calendarDate, {
+            weekday: 'long',
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+        });
+
+        // TODO: Translate "Today" and "selected"
+        const todayTitle = isToday ? 'Today, ' : '';
+        const selectedTitle = isToday ? ' selected' : '';
+
+        const title: string = this._capitalFirstLetter(
+            `${todayTitle}${currentDayTitle}${selectedTitle}`
+        );
+
+        return html`
+            <td
+                role="gridcell"
+                class="spectrum-Calendar-tableCell"
+                title=${title}
+                tabindex=${ifDefined(!isOutsideMonth ? '-1' : undefined)}
+                aria-disabled=${isOutsideMonth || this.disabled}
+                aria-selected=${isSelected}
+            >
+                <span
+                    role="presentation"
+                    class="spectrum-Calendar-date ${classMap(dayClasses)}"
+                    @click=${() => this.handleDayClick(calendarDate)}
+                >
+                    ${this._formatNumber(calendarDate.day)}
+                </span>
+            </td>
+        `;
+    }
+
+    public handlePreviousMonth(): void {
+        this._currentDate = startOfMonth(this._currentDate).subtract({
+            months: 1,
+        });
+    }
+
+    public handleNextMonth(): void {
+        this._currentDate = startOfMonth(this._currentDate).add({ months: 1 });
+    }
+
+    public handleDayClick(calendarDate: CalendarDate): void {
+        this.selectedDate = calendarDate.toDate(this._timeZone);
+        this._setCurrentCalendarDate();
+
+        this.dispatchEvent(
+            new CustomEvent('change', {
+                bubbles: true,
+                cancelable: true,
```

Comment:
You've made this `cancelable` but don't seem to handle the case where the consumer `preventDefault()`s the event.

Reply:
I didn't understand. Could you explain to me better what you mean? Are there any examples I could use as a reference?

Reply:
if the user does event.defaultPrevented how are you handling this?